### ISSUE TITLE
Bug report: Small typo in installlation line in README

### ISSUE THREAD

**Issue by vikjam** [hrecht/censusapi#60]
**Describe the bug**
Small typo in README. There's an extra backtick in the installation line.

Currently,
```
install.packages("censusapi")`
```
Should be,
```
install.packages("censusapi")
```

**To Reproduce**
NA

**Expected behavior**
NA

**R session information:**
NA

**Additional context**
NA

**Comment by hrecht** [comment]
Thanks, will fix in the next version

**Comment by hrecht** [comment]
Thanks for flagging, this is fixed in development version 0.7.0.
